pkg/kubelet/image_manager_test.go

```diff
@@ -37,6 +37,7 @@ func newRealImageManager(policy ImageGCPolicy) (*realImageManager, *container.Fa
 	return &realImageManager{
 		runtime:      fakeRuntime,
 		policy:       policy,
+		minAge:       0,
```

Comment:
There was already a clock object under util. I would prefer not to grab a new dependency without a great reason.